Mathlib/Topology/SubsetProperties.lean

```diff
@@ -1316,6 +1432,28 @@ class SigmaCompactSpace (α : Type*) [TopologicalSpace α] : Prop where
   exists_compact_covering : ∃ K : ℕ → Set α, (∀ n, IsCompact (K n)) ∧ ⋃ n, K n = univ
```

Comment:
For consistency with `CompactSpace`, `ConnectedSpace` and so on, could you replace the definition to be literally `IsSigmaCompact (univ : Set α)` and then prove that it's equivalent to the current definition?